app/src/main/res/raw/changelog.yml

```
@@ -1,4 +1,14 @@
-34.0-beta1: |
+v34.0-beta1: |
+  <h3>🚌🚴 Quest presets</h3>
+  <ul>
+  <li>Show quests that are quick and easy to solve (from a distance) first by default. This makes
+      contributing more efficient and diverse while also easier to do f.e. as a passenger (#2944, ...)</li>
+  <li>You can now save different quest presets for different situations, f.e. one preset when you
```

```suggestion
      contributing more efficient and diverse while also easier to do as e.g. a passenger (#2944, ...)</li>
  <li>You can now save different quest presets for different situations, e.g. one preset when you
```

I've personally never heard of the abbreviation `f.e.`, but only `e.g.`. Apparently it is a thing though: https://www.quora.com/What-does-the-abbreviation-f-e-mean

What's your opinion on this @peternewman @smichel17?

---

Oh, I didn't know that f.e. was so unknown / unused, so I have done it wrong my whole life. Thanks! Gonna do a string-replace...